code/src/App.js

```diff
@@ -1,9 +1,85 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { Summary } from './Summary.js'
+import { Happiness } from './Happiness'
+import { Input } from './Input'
+import { Dropdown } from './Dropdown'
 
 export const App = () => {
+  const [happiness, setHappiness] = useState('')
+  const [feeling, setFeeling] = useState('')
+  const [showResult, setShowResult] = useState(false)
+  const [question, setQuestion] = useState('feeling')
+  const [treatment, setTreatment] = useState("");
+  
+
+  const handleSubmit = (event) => {
+   event.preventDefault() //spara informationen så att sidan inte laddas om (det vanliga sidobeteendet)
+    setShowResult(true)
+  }
+
+  const handleContinueClick = () => {
+    console.log('click continue')
+    if (question === 'feeling') {
+      setQuestion('happiness') 
+    } else if (question === 'happiness') {
+        setQuestion('treatment')
+      
+    }
+  }
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+  <div>
+    {showResult && (
+      <Summary feeling={feeling} happiness={happiness} />
+    )}
+
+    {!showResult && (
+      <form onSubmit={handleSubmit}>
```

Comment:
This `handleSubmit` happens after the first question, if you press the enter key. (which is the same as submitting your form).
See my suggested change above on line 15-26